File: 디자인패턴/java 디자인 패턴/Bridge/Publication.java
public class Publication extends Draft {
    private String publisher;
    private int cost;

    public Publication(String title, String author, String[] content,
        String publisher, int cost) {
        super(title, author, content);
        this.publisher = publisher;
        this.cost = cost;
    }

    public Publication(Draft draft, String publisher, int cost) {
        super(draft.getTitle(), draft.getAuthor(), draft.getContent());
        this.publisher = publisher;
        this.cost = cost;
    }

    @Override
    public void display(Display display) {
        super.display(display);
        System.out.println("#" + publisher + " $" + cost);
        System.out.println();
    }
}

File: 디자인패턴/Python 디자인 패턴/Bridge/Publication.py
from Draft import*
class Publication(Draft):
    def __init__(this, title:str, author:str, content:list, publisher:str, cost:int) -> None:
        this.title = title
        this.author = author
        this.content = content
        this.publisher = publisher
        this.cost = cost
        pass
    
    def __init__(this, darft:Draft, publisher:str, cost:int) -> None:
        super().__init__(title=darft.getTitle(), author=darft.getAuthor(), content=darft.getContent())
        this.publisher = publisher
        this.cost = cost
        pass

    def display(this, display: Display) -> None:
        super().display(display=display)
        print(f"#{this.publisher}, ${this.cost}")
    pass